packages/repository-json-schema/src/__tests__/unit/filter-json-schema.unit.ts

```diff
@@ -127,6 +129,50 @@ describe('getFilterJsonSchemaFor', () => {
     ]);
   });
 
+  it('non-strict model should allow any value in "fields"', () => {
```

Comment:
We write test names to read like an English sentence, e.g. `it disallows "include"` for the test above.

Can you please improve names of the new test to follow that same style too?

For example:

```suggestion
  it('allows free-form properties in "fields" for non-strict models', () => {
```